src/main/java/ama/post/Post.java

```diff
@@ -36,5 +37,19 @@ public void setText(String text) {
         this.text = text;
     }
 
+    public User getUser() {return op;}
+
+    public void setUser(User user) {
+        this.op = user;
+    }
+
     public Long getId() { return id; }
+
+    public Collection<CommentPost> getReplies() {
+        return replies;
+    }
+
+    public void setReplies(Collection<CommentPost> replies) {
```

Comment:
Yep. Since this isn't actually a new relationship, we can have it map to context and then access 'context' from a leaf node or 'replies' from a parent node